hw06-ATM/src/main/java/homework/CashBox.java

```diff
@@ -0,0 +1,28 @@
+package homework;
+
+import java.util.Objects;
+
+public class CashBox {
+    private final Integer numberNominal;
```

Comment:
```suggestion
    private final Nominals numberNominal;
```

Можно прямо перечисление использовать как ключ